go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dogeorg/dogeboxd
 
-go 1.22.2
+go 1.23.0
```

Comment:
Hmm, this may require changes elsewhere in the build process?

Reply:
Ah, okay. Reason for it is to get `maps.Values`

Reply:
From a quick glance it _looks_ like nix `24.11` ships `1.23.7`, so this should be OK actually